### ISSUE TITLE
Cannot find `metadata.rb` when using specifying `cookbook.metadata` in `Buildfile`.

### ISSUE THREAD

**Issue by athompson-r7** [rapid7/builderator#65]
Cannot find `metadata.rb` when using specifying `cookbook.metadata` in `Buildfile`.

```
An error occurred while reading the Berksfile:

  Could not open or read: '/Users/athompson/code/chef/chef-rapid7-ruby/.builderator/metadata.rb'
```

**Comment by jmanero-r7** [comment]
Yeah, that feature's never going to work because of the way builderator wraps Berkshelf now. The `metadata` Berksfile method instructs Berkshelf to look in the same directory as the Berksfile for a metadata.rb. Because we generate a Berksfile in a sub-directory, there'll never be a metadata.rb file to load.

Instead, add a cookbook with a relative path.

**Comment by athompson-r7** [comment]
@jmanero-r7 was this fixed by #68?

**Comment by athompson-r7** [comment]
Closing this
